modules/installation-and-upgrade/pages/container-deployment/suma/server-deployment-suma.adoc

```diff
@@ -198,9 +214,9 @@ transactional-update register -p SUSE-Manager-Server/5.0/x86_64 -r <reg_code>
 . Reboot.
 
 
-=== Update the System
+==== Update the System
 
-.Procedure
+.Procedure: Updating the System
 . Log in as *root*.
 . Run **transactional-update**:
```

Comment:
Just out of curiosity, why do we not mention to install `mgradm` and `mgradm-bash-completion` here, like we do for the proxy?
<img width="1003" alt="image" src="https://github.com/user-attachments/assets/85fcb9da-eaf8-4300-abf7-d81c0b0721a8" />
<img width="1003" alt="image" src="https://github.com/user-attachments/assets/61f1e7f5-2ae5-425d-8438-ddb3347ab050" />

Reply:
I think it now will get installed automatically.  Trying to ping @mcalmer for clarification: Do we still need to install `mgradm` and completion tools with the first `transactional-update` run after preparing Micro as the server host OS? On both, 5.0.4 and 5.1.x?